Header header

int16 encoder3
int16 encoder4
int16 encoder_step1
int16 encoder_step2
int16 DAC_value1
int16 DAC_value2

uint8  step1_finish_flag
uint8  step2_finish_flag
